### PR TITLE
scope and dao save fixes.

### PR DESCRIPTION
**Pull request by jaycarlton**
A series of "improvements" and "fixes" to this test case. Still running into some trouble with the transient fieldls on DbParticipantCohortStatus. I was trying to make sure the reference passed in to `save()` wasn't reused.

### CODE REVIEW

File: api/src/test/java/org/pmiops/workbench/api/CohortReviewControllerTest.java [hunk 1]
```diff
@@ -282,82 +288,75 @@ public void setUp() {
...
-            .ethnicityConceptId(TestDemo.NOT_HISPANIC.getConceptId())
-            .ethnicity(TestDemo.NOT_HISPANIC.getName())
-            .birthDate(new java.sql.Date(today.getTime()))
-            .deceased(false);
-
-    participantCohortStatusDao.save(participantCohortStatus1);
-    participantCohortStatusDao.save(participantCohortStatus2);
-
-    stringAnnotationDefinition =
+    participantCohortStatus1 = participantCohortStatusDao.save(new DbParticipantCohortStatus()
```
Comment (jaycarlton) [inline api/src/test/java/org/pmiops/workbench/api/CohortReviewControllerTest.java]: In the previous code, we manually assigned the transient (but member-backed) fields for gender and race, and a couple of others. After changing to only assign the `participantCohortStatusX` members after `save()`, I notice that these guys are still null. I'd expected them to be calculated based on other information in the model if they're truly transient.

File: api/src/test/java/org/pmiops/workbench/api/CohortReviewControllerTest.java [hunk 2]
```diff
@@ -283,81 +299,78 @@ public void setUp() {
...
-            .race(TestDemo.ASIAN.getName())
-            .ethnicityConceptId(TestDemo.NOT_HISPANIC.getConceptId())
-            .ethnicity(TestDemo.NOT_HISPANIC.getName())
-            .birthDate(new java.sql.Date(today.getTime()))
-            .deceased(false);
+        participantCohortStatusDao.save(
+            new DbParticipantCohortStatus()
+                .statusEnum(CohortStatus.NOT_REVIEWED)
+                .participantKey(key1)
+                .genderConceptId(TestDemo.MALE.getConceptId())
```
Comment (jaycarlton) [inline api/src/test/java/org/pmiops/workbench/api/CohortReviewControllerTest.java]: Maybe rename TestDemo to `TestConcepts`.

Comment (freemabd) [reply api/src/test/java/org/pmiops/workbench/api/CohortReviewControllerTest.java]: Done.

**Comment by jaycarlton** [comment]
replaced by #2922 as Brian has graciously taken over the branch.

**Pull request closed without merging**